examples/univer-sheet-ts/src/main.tsx

```diff
@@ -1,20 +1,32 @@
-import { DocPlugin } from '@univerjs/base-docs';
+import { DocPlugin, zh as DocPluginZh } from '@univerjs/base-docs';
 import { RenderEngine } from '@univerjs/base-render';
-import { SheetPlugin } from '@univerjs/base-sheets';
-import { UIPlugin } from '@univerjs/base-ui';
+import { SheetPlugin, zh as SheetPluginZh } from '@univerjs/base-sheets';
+import { UIPlugin, zh as UIPluginZh } from '@univerjs/base-ui';
 import { DEFAULT_WORKBOOK_DATA_DEMO } from '@univerjs/common-plugin-data';
 import { LocaleType, Univer } from '@univerjs/core';
-import { FindPlugin } from '@univerjs/sheets-plugin-find';
+import { FindPlugin, zh as FindPluginZh } from '@univerjs/sheets-plugin-find';
 import { DEFAULT_FORMULA_DATA_DEMO, FormulaPlugin } from '@univerjs/sheets-plugin-formula';
 import { ImagePlugin } from '@univerjs/sheets-plugin-image';
-import { ImportXlsxPlugin } from '@univerjs/sheets-plugin-import-xlsx';
-import { NumfmtPlugin } from '@univerjs/sheets-plugin-numfmt';
-import { DocUIPlugin } from '@univerjs/ui-plugin-docs';
-import { SheetUIPlugin } from '@univerjs/ui-plugin-sheets';
+import { ImportXlsxPlugin, zh as ImportXlsxPluginZh } from '@univerjs/sheets-plugin-import-xlsx';
+import { NumfmtPlugin, zh as NumberfmtPluginZh } from '@univerjs/sheets-plugin-numfmt';
+import { DocUIPlugin, zh as DocUIPluginZh } from '@univerjs/ui-plugin-docs';
+import { SheetUIPlugin, zh as SheetUIPluginZh } from '@univerjs/ui-plugin-sheets';
 
 // univer
 const univer = new Univer({
     locale: LocaleType.EN,
+    locales: {
+        zh: {
```

Comment:
IHMO it is better to extract these into another file? Would be a practice for our users.